recipes/islandpath/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "v1.0.3" %}
+
+package:
+  name: islandpath
+  version: {{version}}
+
+source:
+  #  git_url: https://github.com/brinkmanlab/islandpath
+  url: https://github.com/brinkmanlab/islandpath/archive/{{version}}.tar.gz
+  sha256: ac9198240fe1674b876fc0ab4c461e5a34799ddba521481eb2bb15324c2845bd
+  
+requirements:
+  build:
+    - perl-app-cpanminus
+    - perl-moose
+    - make
+  run:
+    - hmmer
+    - perl-bioperl 
+    - perl-data-dumper 
+    - perl-log-log4perl 
+    - perl-config-simple 
+    - perl-moose 
+    #- perl-moosex-singleton
+
+test:
+  source_files:
```

Comment:
This is currently not supported in Bioconda, you can not user a file as test.

Reply:
Removed